docs/api.asciidoc

```diff
@@ -0,0 +1,191 @@
+[[api]]
+== Public API
+
+The Elastic APM Python agent has several public APIs.
+Most of these APIs are not needed when using one of our supported frameworks (Django and Flask),
```

Comment:
How about changing this to something like `Most of the public API functionality is not needed when.. `